holoviews/plotting/mpl/element.py

```diff
@@ -711,40 +749,3 @@ def update_frame(self, key, ranges=None, element=None):
             self._adjust_legend(element, axis)
 
         self._finalize_axis(key, ranges=ranges)
-
-
-
-class DrawPlot(ElementPlot):
```

Comment:
Basically all you do now is define a ``teardown_handles`` method containing:

```
if self.zorder == 0: self.handles['axis'].cla()
```

and then define a ``get_data`` and ``init_artist`` method, if you ever figure out how to update the plot by changing an existing artist you just replace the ``teardown_handles`` method with an ``update_handles`` method and it will use that instead.